test/test_msec.py

```diff
@@ -1,59 +1,64 @@
+            definition="NBA team based in Chicago, IL",
+            type="team"
+        )
 
-        section_bulls2  = odml.Section(
-                                       name="Chicago_City_Pub",
-                                       definition="NBA team pub",
-                                       type="pub"
-                                      )
+        section_bulls2 = odml.Section(
+            name="Chicago_City_Pub",
+            definition="NBA team pub",
+            type="pub"
+        )
 
         section_bull3 = odml.Section(
-                                     name="Bill_Clinton",
-                                     definition="President supporter",
-                                     type="fan"
-                                    )
+            name="Bill_Clinton",
+            definition="President supporter",
+            type="fan"
+        )
 
         prop1 = odml.Property(
-                              name="Coach",
-                              value="Phil Jackson",
-                              definition="Run the team."
-                             )
-        
+            name="Coach",
+            value="Phil Jackson",
+            definition="Run the team."
+        )
+
         prop2 = odml.Property(
-                              name="Euro_player",
-                              value="Tony Kukoc",
-                              definition="European Player"
-                             )
+            name="Euro_player",
+            value="Tony Kukoc",
+            definition="European Player"
+        )
 
         section_bulls.append(
-                             section_bulls2,
-                             section_bull3,
-                             prop1,
-                             prop2
-                            )
-        
+            section_bulls2,
+            section_bull3,
+            prop1,
+            prop2
+        )
+
         section_bulls0.append(section_bulls2)
         section_bulls0.append(section_bull3)
         section_bulls0.append(prop1)
         section_bulls0.append(prop2)
-         
-        self.assertTrue(section_bulls.sections[0].name == section_bulls0.sections[0].name)
-        self.assertTrue(section_bulls.sections[1].name == section_bulls0.sections[1].name)
-        self.assertTrue(section_bulls.properties[0].name == section_bulls0.properties[0].name)
-        self.assertTrue(section_bulls.properties[1].name == section_bulls0.properties[1].name)
\ No newline at end of file
+
+        self.assertTrue(section_bulls.sections[
+                        0].name == section_bulls0.sections[0].name)
+        self.assertTrue(section_bulls.sections[
+                        1].name == section_bulls0.sections[1].name)
+        self.assertTrue(section_bulls.properties[
+                        0].name == section_bulls0.properties[0].name)
+        self.assertTrue(section_bulls.properties[
+                        1].name == section_bulls0.properties[1].name)
```

Comment:
Ouch .. These look horrible !! :sweat_smile: .. 

I think the latter approach, i.e. breaking on `==` , will look better, because some of the expressions are still greater than required length in case of the former one.
Will fix this .. :+1: